index.html

```diff
@@ -1,299 +1,312 @@
+              <p class="prgph">
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.
+              </p>
+              <a href="#" class="connect">LET'S CONNECT</a>
+
+              <ul class="media-platform">
+                <li>
+                  <a href="#"
+                    ><img src="./icons/twitter.svg" alt="twitter"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./icons/Linkedin-icon.svg" alt="Linkedin"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"> <img src="./icons/medium.svg" alt="medium" /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./icons/github.svg" alt="github" /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./icons/Vector.svg" alt="Vector" /></a>
+                </li>
+              </ul>
+              <button type="button" class="btn-get">Get my resume</button>
+            </div>
+            <div class="bottom">
+              <ul class="lang-li">
+                <li class="">
+                  <div class="lang-name">
+                    <h2>Languages</h2>
+                    <span
+                      ><img src="./icons/Uniondown.png" alt="down-arrow"
+                    /></span>
+                  </div>
+                  <ul class="classic languages-ul">
+                    <li>
+                      <img src="./photos/Ellipse4.png" alt="js" />
+                      <h3 class="classic-name">Javascript</h3>
+                    </li>
+                    <li>
+                      <img src="./photos/Ellipse1.png" alt="html" />
+                      <h3 class="classic-name">HTML</h3>
+                    </li>
+                    <li>
+                      <img src="./photos/Ellipse2.png" alt="css" />
+                      <h3 class="classic-name">CSS</h3>
+                    </li>
+                  </ul>
+                </li>
+                <li class="common-li">
+                  <div class="lang-name">
+                    <h2>Frameworks</h2>
+                    <span
+                      ><img src="./icons/Uniondown.png" alt="down-arrow"
+                    /></span>
+                  </div>
+                </li>
+                <li class="common-li">
+                  <div class="lang-name">
+                    <h2>Skills</h2>
+                    <span
+                      ><img src="./icons/Uniondown.png" alt="down-arrow"
+                    /></span>
+                  </div>
+                </li>
+
+                <li></li>
+              </ul>
+            </div>

```

Comment:
- [ ] Kindly make the head div 40% and bottom div 60% kindly check the Figma design again, see screenshots below 👍🏿 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![wallla](https://user-images.githubusercontent.com/69638013/154819420-dadf19e6-6cb7-48c6-adb4-67f9aefc2582.PNG))  | ![hahaha](https://user-images.githubusercontent.com/69638013/154819427-a634c03d-12fe-454d-9a38-c63e9cd15368.PNG)

Reply:
done